packages/Card/src/components/Text/Text.styles.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+
+export const textStyles = styled.p`
+  color: ${tokens.color.black};
+  ${stylers.fontSize(-2)};
+  height: 55px;
```

Comment:
Should this be 56, which is divisible by 8? and therefore use the `spacer` helper